shared/studio/tabs/auth/state/index.tsx

```diff
@@ -1225,7 +1226,7 @@ export class DraftSMTPConfig
             if (typeof val === "string") {
               val = val.trim();
             }
-            if (val == null || val == "") {
+            if (val == null || val === "") {
```

Comment:
Not a blocker, but it seems like we could just return `""` in this case, and drop the later filter?